appshell/appshell_extensions_win.cpp

```diff
@@ -422,66 +422,35 @@ int32 ShowOpenDialog(bool allowMultipleSelection,
     ConvertToNativePath(initialDirectory);
 
     if (chooseDirectory) {
-        // check current OS version
-        OSVERSIONINFO osvi;
-        memset(&osvi, 0, sizeof(OSVERSIONINFO));
-        osvi.dwOSVersionInfoSize = sizeof(OSVERSIONINFO);
-        if (GetVersionEx(&osvi) && (osvi.dwMajorVersion >= 6)) {
-            // for Vista or later, use the MSDN-preferred implementation of the Open File dialog in pick folders mode
-            IFileDialog *pfd;
-            if (SUCCEEDED(CoCreateInstance(CLSID_FileOpenDialog, NULL, CLSCTX_INPROC_SERVER, IID_PPV_ARGS(&pfd)))) {
-                // configure the dialog to Select Folders only
-                DWORD dwOptions;
-                if (SUCCEEDED(pfd->GetOptions(&dwOptions))) {
-                    pfd->SetOptions(dwOptions | FOS_PICKFOLDERS | FOS_DONTADDTORECENT);
-                    IShellItem *shellItem = NULL;
-                    if (SUCCEEDED(SHCreateItemFromParsingName(initialDirectory.c_str(), 0, IID_IShellItem, reinterpret_cast<void**>(&shellItem))))
-                        pfd->SetFolder(shellItem);
-                    pfd->SetTitle(title.c_str());
-                    if (SUCCEEDED(pfd->Show(GetActiveWindow()))) {
-                        IShellItem *psi;
-                        if (SUCCEEDED(pfd->GetResult(&psi))) {
-                            LPWSTR lpwszName = NULL;
-                            if(SUCCEEDED(psi->GetDisplayName(SIGDN_DESKTOPABSOLUTEPARSING, (LPWSTR*)&lpwszName))) {
-                                // Add directory path to the result
-                                std::wstring wstrName(lpwszName);
-                                ExtensionString pathName(wstrName);
-                                ConvertToUnixPath(pathName);
-                                selectedFiles->SetString(0, pathName);
-                                ::CoTaskMemFree(lpwszName);
-                            }
-                            psi->Release();
+        IFileDialog *pfd;
+        if (SUCCEEDED(CoCreateInstance(CLSID_FileOpenDialog, NULL, CLSCTX_INPROC_SERVER, IID_PPV_ARGS(&pfd)))) {
```

Comment:
If you add `w=1` to an url on github you'll not see the whitespaces changes.
https://github.com/adobe/brackets-shell/pull/549/files?w=1
It can be useful to times like this